api/schemas/flow-video.json

```diff
@@ -137,12 +143,16 @@
                             ]
                         },
                         "horiz_chroma_subs": {
-                            "description": "Horizontal chroma component sub-sampling.",
-                            "type": "integer"
+                            "description": "Horizontal chroma component sub-sampling. When unc_type is set to a YUV type, horiz_chroma_subs must be set.",
+                            "type": "integer",
+                            "exclusiveMinimum": 0,
+                            "maximum": 2
```

Comment:
Not sure whether this limit needs or should be set unless we are restricting the formats supported. Wikipedia for example [lists 4:1:0 and 3:1:1](https://en.wikipedia.org/wiki/Chroma_subsampling)

Reply:
Fair. I'll remove the max and set the minimum to inclusive 0.

Reply:
I think the minimum should be 1, i.e. no-sub-sampling.

Reply:
Bah. 4:2:0 confusion strikes again